modules/nodes-cluster-overcommit-node-memory.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-overcommit.adoc
+
+[id='nodes-cluster-overcommit-node-memory_{context}']
+
+= Reserving memory across quality of service tiers
+
+You can use the `experimental-qos-reserved` parameter to specify a percentage of memory to be reserved
```

Comment:
this became not `experimental` upstream at some point.  not sure which release.

Reply:
@sjenning Is this the change you mean? Is the upstream change 3.12+ ?